terraform/aks/workspace_variables/qa_aks.tfvars.json

```diff
@@ -15,5 +15,31 @@
   "webapp_replicas": 2,
   "worker_replicas": 1,
   "secondary_worker_replicas": 1,
-  "clock_worker_replicas": 1
+  "clock_worker_replicas": 1,
+  "statuscake_alerts": {
+    "apply-aks-qa": {
+      "website_name": "Apply-Teacher-Training-AKS-QA",
+      "website_url": "https://qa2.apply-for-teacher-training.service.gov.uk/integrations/monitoring/all",
+      "check_rate": 30,
+      "contact_group": [
+        282783
+      ]
+    },
+    "apply-aks-qa-check": {
+      "website_name": "Apply-Teacher-Training-Check-AKS-QA",
+      "website_url": "https://qa2.apply-for-teacher-training.service.gov.uk/check",
+      "check_rate": 30,
+      "contact_group": [
+        282783
+      ]
+    },
+    "apply-aks-cloudapps-qa": {
+      "website_name": "Apply-Teacher-Training-Cloudapps-AKS-QA",
```

Comment:
"Cloudapps" does not make much sense anymore. Maybe "Direct" or "Ingress"... ?

Reply:
changed to Ingress